doc/api/assert.md

```diff
@@ -1249,10 +1266,8 @@ second argument. This might lead to difficult-to-spot errors.
 [`ERR_INVALID_RETURN_VALUE`]: errors.html#errors_err_invalid_return_value
 [`Error.captureStackTrace`]: errors.html#errors_error_capturestacktrace_targetobject_constructoropt
 [`Error`]: errors.html#errors_class_error
-[`Map`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
```

Comment:
Why is this being removed? `Map` and `Set` are mentioned above.

Reply:
I removed the only two entries which were linked. Shall I change the existing entries to add the link?

Reply:
I referenced all entries now.